test/jdk/javax/net/ssl/templates/TLSBase.java

```diff
@@ -97,8 +97,8 @@ public void write(SSLSocket sock, byte[] data) throws Exception {
     private static KeyManager[] getKeyManager(boolean empty) throws Exception {
         FileInputStream fis = null;
         if (!empty) {
-            fis = new FileInputStream(System.getProperty("test.src", "./") + "/" + pathToStores +
-                "/" + keyStoreFile);
+            fis = new FileInputStream(System.getProperty("test.src", "./") +
+                "/" + pathToStores + "/" + keyStoreFile);
```

Comment:
My Java style guide says to break before the operator.

Reply:
I have never heard that.  I have always ended with a plus and I believe everyone else does too if I'm not mistaken.

Reply:
https://cr.openjdk.org/~alundblad/styleguide/index-v6.html#toc-constants

Probably showed up in a code review comment from someone.

Reply:
There are a lot of places where that style is violated.  Looking at just a few random files I didn't see any following the correct style (no scientific study).  I'm going to leave it as is.